lib/openstudio/aws/aws.rb

```diff
@@ -85,10 +87,10 @@ def initialize(options = {})
       #
       # end
 
-      # command line call to create a new instance.  This should be more tightly integrated with teh os-aws.rb gem
-      def create_server(options = {}, instances_filename = 'server_data.json')
+      # command line call to create a new instance.  This should be more tightly integrated with the os-aws.rb gem
+      def create_server(options = {})
         defaults = {
-          instance_type: 'm2.xlarge',
+          instance_type: 'm3.xlarge',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.